Makefile

```diff
@@ -0,0 +1,13 @@
+all: build
+
+build:
+	@docker build --tag=joeybaker/syncthing:latest .
+
+base:
+	@docker pull ubuntu:14.04
+
+rebuild: base
+	@docker build --tag=joeybaker/syncthing:latest .
+
+release: rebuild
+	@docker build --tag=joeybaker/syncthing:0.12.11 .
```

Comment:
Yes it does, maybe there is a clever way to use environment variables from the build?